src/store/metadata/stub/content/editions/versions/local-authority-greenhouse-gas-emissions_2023-06.json

```diff
@@ -0,0 +1,76 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions",
+    "@type": "hydra:Collection",
+    "title": "Local authority greenhouse gas emissions",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions/1",
+            "identifier": "1",
+            "issued": "2023-06-29T00:00:00Z",
+            "title": "Local authority greenhouse gas emissions",
+            "summary": "UK local authority estimates of greenhouse gas emissions",
+            "description": "These statistics provide a breakdown of greenhouse gas emissions across the UK, using...",
+            "download_url": "https://staging.idpd.uk/datasets/local-authority-greenhouse-gas-emissions/editions/2023-06/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
```

Comment:
Defined columns don't quite match with the dataset. Upper and lower confidence intervals are missing here, as is observation_status.
A country column is defined here, but the dataset doesn't include any countries. The dataset just uses areanm, areacd, and geography.

Reply:
Yes, this is a peculiar case, this was a temporary work to get the demo to work, then. 

The existing dataset was uploaded then as the tidy csv for the climate-change is quite large and is difficult to push to the remote branch. We will probably delete the test dataset csv while figuring out how to upload the csv.